src/Contact.Api/Program.cs

```diff
@@ -12,6 +13,8 @@
 builder.Services.AddDbContext<AppDbContext>(options =>
     options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
 
+builder.Services.AddAutoRegister<AppDbContext>();
```

Comment:
AddAutoRegister is registering repository and services implementations to the respective interfaces... Using for DependencyInjection.